cannon/mipsevm/tests/evm_common_test.go

```diff
@@ -315,6 +315,69 @@ func TestEVMSingleStep_LoadStore(t *testing.T) {
 	}
 }
 
+func TestEVMSingleStep_MovzMovn(t *testing.T) {
```

Comment:
what is the reason to change EVM test?

Reply:
ehhh this PR is prob just targeting the wrong commit. :\
I didn't change this.